spanner/sessionclient.go

```diff
@@ -230,11 +248,29 @@ func (sc *sessionClient) executeBatchCreateSessions(client *vkit.Client, createC
 			consumer.sessionCreationFailed(ToSpannerError(ctx.Err()), remainingCreateCount)
 			break
 		}
+		var mdForGFELatency metadata.MD
 		response, err := client.BatchCreateSessions(ctx, &sppb.BatchCreateSessionsRequest{
 			SessionCount:    remainingCreateCount,
 			Database:        sc.database,
 			SessionTemplate: &sppb.Session{Labels: labels},
-		})
+		}, gax.WithGRPCOptions(grpc.Header(&mdForGFELatency)))
+
+		if GFELatencyOrHeaderMissingCountEnabled && md != nil{
```

Comment:
This if statement should check that `mdForGFELatency != nil`:

```suggestion
		if GFELatencyOrHeaderMissingCountEnabled && mdForGFELatency != nil{
```